extension/intents/clipboard/contentScript.js

```diff
@@ -65,7 +65,7 @@ this.contentScript = (function() {
 
   communicate.register("copy", message => {
     const { copyType } = message;
-    const result = types[copyType]();
+    const result = types[copyType]() || message.value;
```

Comment:
I'd rather see this special case in `clipboard.js`. Or really `clipboard.copyValue` could just use something besides `copy()` to copy the text. I think you could call `navigator.clipboard.writeText()` directly from the intent handler.